src/fragments/lib/auth/js/customui.mdx

```diff
@@ -1,5 +1,19 @@
 ### Customize UI theme
 
+<Callout warning={true}>
```

Comment:
Should we put this in a separate fragment like https://github.com/aws-amplify/docs/blob/2e9afd481cd22f2c22bade93b61e850c1c18f592/src/fragments/ui/auth/developer-preview-callout-js.mdx